lib/ui/platform_dispatcher.dart

```diff
@@ -801,11 +801,46 @@ class PlatformDispatcher {
   ///
   ///  * [SchedulerBinding], the Flutter framework class which manages the
   ///    scheduling of frames.
+  ///  * [scheduleWarmUpFrame], which should only be used to schedule warm up
+  ///    frames.
   void scheduleFrame() => _scheduleFrame();
 
   @Native<Void Function()>(symbol: 'PlatformConfigurationNativeApi::ScheduleFrame')
   external static void _scheduleFrame();
 
+  /// Schedule a frame to run as soon as possible, rather than waiting for the
+  /// engine to request a frame in response to a system "Vsync" signal.
+  ///
+  /// This method is used during application startup so that the first frame
```

Comment:
This phrasing is unclear from the standpoint of an author of an app or framework that does not use the standard Flutter framework (e.g. a game engine, or a framework with a different design). What does "is used" mean? Used by what?

Reply:
This paragraph is copied from `SchedulerBinding.scheduleWarmUpFrame`, but I agree with your concern. Do you think the following way would be better?

```
  /// The application can call this method as soon as it starts up so that the first
  /// frame (which is likely to be quite expensive) can start a few extra
  /// milliseconds earlier. [...]
```

Reply:
Yep, SGTM!